backend/src/index.js

```diff
@@ -2,7 +2,9 @@ const express = require('express')
 const bodyParser = require('body-parser')
 const mongoose = require('mongoose')
 const { Candidate, Stats, Match } = require('./models')
-const { errorWrap, getStats } = require('./utils')
+const { getStats } = require('./utils')
+const { errorWrap } = require('./middleware')
+const errorHandler = require('./middleware/errorHandler')
```

Comment:
Couldn't this line be removed and line 6 be replaced with: `const {errorWrap, errorHandler } = require('./middleware')`?